app/components/PostcodeMap.tsx

```diff
@@ -1,76 +1,69 @@
-// @flow
-import React from 'react';
+import React from "react";
 import { MapContainer } from 'react-leaflet/MapContainer';
 import { GeoJSON } from 'react-leaflet/GeoJSON';
 import { TileLayer } from 'react-leaflet/TileLayer';
 import { Marker } from 'react-leaflet/Marker';
 import { useMapEvent } from 'react-leaflet/hooks';
-import { divIcon, type Layer } from 'leaflet';
-import { feature } from 'topojson';
-import topoData from '../map/berlin-postcodes-data.topo';
-import labels from '../map/labels';
-
+import type { Layer } from "leaflet";
+import { divIcon } from "leaflet";
+import { feature } from "topojson";
+import topoData from "../map/berlin-postcodes-data.topo";
+import labels from "../map/labels";
```

Comment:
newline after imports

Reply:
noted down in the prettier issue

Reply:
I'm not sure if prettier can *add* newlines in this case. The PR should not make the formatting worse, so we might need to tackle this manually.